src/NexusMods.App.UI/Controls/GameWidget/GameWidget.axaml.cs

```diff
@@ -50,27 +46,25 @@ public GameWidget()
                 
                 this.BindCommand(ViewModel, vm => vm.ViewGameCommand, v => v.ViewGameButton)
                     .DisposeWith(d);
+                
+                StoreIcon.Classes.Add(MapGameStoreToIconClass(ViewModel!.Installation.Store));
             }
         );
     }
 
-    private static Bitmap BlurAvaloniaImage(Bitmap bitmap)
+    /// <summary>
+    /// Returns a game store icon class for a given game store. The plain text store and the icon class name aren't always the same.
+    /// </summary>
+    /// <param name="store">A <see cref="GameStore"/> object</param>
+    /// <returns>An icon representing the game store or a question mark icon if not found.</returns>
+    private string MapGameStoreToIconClass(GameStore store) => store.Value switch
     {
-        using var inputSkBitmap = bitmap.ToSkiaBitmap();
-        using var outputSkBitmap = new SKBitmap(inputSkBitmap.Info, SKBitmapAllocFlags.ZeroPixels);
-
-        using (var skCanvas = new SKCanvas(outputSkBitmap))
-        using (var skPaint = new SKPaint())
-        {
-            skPaint.ImageFilter = SKImageFilter.CreateBlur(
-                sigmaX: 100f,
-                sigmaY: 100f
-            );
-
-            var skRect = inputSkBitmap.Info.Rect;
-            skCanvas.DrawBitmap(inputSkBitmap, skRect, skPaint);
-        }
-
-        return outputSkBitmap.ToAvaloniaImage();
-    }
+        "Steam" => "Steam",
+        "GOG" => "GOG",
+        "Epic Games Store" => "Epic",
+        "Origin" => "Ubisoft",
+        "EA Desktop" => "EA",
+        "Xbox Game Pass" => "Xbox",
+        _ => "Help",
+    };
```

Comment:
Would be better to have a `MapGameStoreToIconValue` function that takes in a game store and returns an `IconValue`. The icon value should be bind to the UI. Also don't switch on the string value of `GameStore.Value`, just do multiple `if` statements.